### PR TITLE
Add the 2018 entries from Timo's list.

### PR DESCRIPTION
**Pull request by bangerth**
These are all 2018 entries from Timo's list that were found to have used deal.II for their numerical examples.

### CODE REVIEW

File: publications-2018.bib [hunk 1]
```diff
@@ -1484,4 +1484,183 @@ @article{Verner2018
...
+
+@Phdthesis{Alhazmi2018,
+  author = {Alhazmi, M.},
+  title  = {{Exploring mechanisms for pattern formation through coupled bulk-surface PDEs}},
+  year   = {2018},
+  url    = {http://sro.sussex.ac.uk/id/eprint/78232/},
+  School = {University of Sussex}
+}
+
+@Phdthesis{Zhao2018,
```
Comment (tjhei) [inline publications-2018.bib]:
```suggestion
@Phdthesis{Zhao2018b,
```

File: publications-2018.bib [hunk 2]
```diff
@@ -1484,4 +1484,183 @@ @article{Verner2018
...
+@misc{bonito2018finite,
+    title={Finite element approximation of an obstacle problem for a class of integro-differential operators},
+    author={Andrea Bonito and Wenyu Lei and Abner J. Salgado},
+    year={2018},
+    eprint={1808.01576},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+@misc{bonito2018finite,
```
Comment (tjhei) [inline publications-2018.bib]:
```suggestion
@misc{bonito2018strain,
```

File: publications-2018.bib [hunk 3]
```diff
@@ -1484,4 +1484,183 @@ @article{Verner2018
...
+  volume = {9},
+  number = {2},
+  pages = {267--294},
+  author = {Anne Glerum and Cedric Thieulot and Menno Fraters and Constantijn Blom and Wim Spakman},
+  title = {Nonlinear viscoplasticity in {\&}lt$\mathsemicolon$span style={\&}quot$\mathsemicolon${\&}quot$\mathsemicolon$ class={\&}quot$\mathsemicolon$text smallcaps{\&}quot$\mathsemicolon${\&}gt$\mathsemicolon${ASPECT}{\&}lt$\mathsemicolon$/span{\&}gt$\mathsemicolon$: benchmarking and applications to subduction},
+  journal = {Solid Earth}
+}
+
+@Phdthesis{Lorca2018,
+  author = {Lorca, JP Lucero},
```
Comment (tjhei) [inline publications-2018.bib]:
```suggestion
  author = {Lucero Lorca, Jose Pablo},
```

**Comment by tjhei** [comment]
```
WARN - Duplicate entry key: 'Giuliani2018' in file '../publications-2018.bib', skipping ...
WARN - Entry RobergeNorato2018 does not parse correctly
WARN - Duplicate entry key: 'Zhao2018' in file '../publications-2018.bib', skipping ...
WARN - Entry Lorca2018 does not parse correctly
WARN - Duplicate entry key: 'bonito2018finite' in file '../publications-2018.bib', skipping ...
ERROR - BibTeX subsystem: /tmp/Ulbgvqh26w/publications-2018.bib_315.utf8, line 1443, syntax error: found "@article", expected end of entry ("}" or ")") (skipping to next "@")
```

Can you rebase to #147 to fix the last error?

**Comment by bangerth** [comment]
@tjhei: I took your suggestions -- thank you! (Most of the patch was more or less automatically generated, using the DOI and arXiv numbers and letting doi2bib and arxiv.org generate the entries automatically.

File: publications-2018.bib [hunk 4]
```diff
@@ -1485,4 +1485,183 @@ @article{Verner2018
...
+  doi = {10.5194/se-9-267-2018},
+  url = {https://doi.org/10.5194/se-9-267-2018},
+  year = {2018},
+  month = mar,
+  publisher = {Copernicus {GmbH}},
+  volume = {9},
+  number = {2},
+  pages = {267--294},
+  author = {Anne Glerum and Cedric Thieulot and Menno Fraters and Constantijn Blom and Wim Spakman},
+  title = {Nonlinear viscoplasticity in {\&}lt$\mathsemicolon$span style={\&}quot$\mathsemicolon${\&}quot$\mathsemicolon$ class={\&}quot$\mathsemicolon$text smallcaps{\&}quot$\mathsemicolon${\&}gt$\mathsemicolon${ASPECT}{\&}lt$\mathsemicolon$/span{\&}gt$\mathsemicolon$: benchmarking and applications to subduction},
```
Comment (tjhei) [inline publications-2018.bib]:
```suggestion
  title = {Nonlinear viscoplasticity in {ASPECT}: benchmarking and applications to subduction},
```

Comment (bangerth) [reply publications-2018.bib]: Yes, I just found that a few seconds after you :-)

Comment (bangerth) [reply publications-2018.bib]: That took a little while to find in the CI output :-)

Comment (tjhei) [reply publications-2018.bib]: yep, a bit difficult, sadly.

**Pull request merged**